topics/ecology/tutorials/phylogeny-data-prep/tutorial.md

```diff
@@ -0,0 +1,376 @@
+---
+layout: tutorial_hands_on
+
+title: Preparing genomic data for phylogeny reconstruction
+zenodo_link: https://zenodo.org/record/6610704#.Ypn3FzlBw5k
+questions:
+- How do I find a set of common proteins (orthologs) across related species or strains?
+- How do I organize a set of orthologs to infer evolutionary relations between species or strains (phylogenetic reconstruction)?
+objectives:
+- Mask repetitive elements from a genome
+- Annotate (predict protein-coding genes) the genomes of the samples to compare
+- Find a set of common proteins across the samples (orthologs)
+- Align orthologs across samples
+time_estimation: 3H
+key_points:
+- You now are able to
+- Predict proteins in a nucleotide sequence *de-novo* using **funannotate_predict**
+- Find orthologs across different samples with **orthofinder**
+- Align orthologs with **ClustalW** in preparation for phylogeny reconstruction <!-- link to phylogeny reconstruction training. -->
+contributors:
+- roncoronimiguel
+- brigidagallone
```

Comment:
If you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing. All of the sub-fields are optional, so if you just have authors/editors you can do just those two :) 

```
contributions:
  authorship: 
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.